VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -32,101 +33,129 @@
     //UI and animation time need to get from production team.
     private boolean enableFlyinWindow = false;
```

Comment:
I *think* it's the overlay from the top we show with errors. But if it's not being used and those exist elsewhere, then we probably don't need it.